sql/hive/src/main/scala/org/apache/spark/sql/hive/client/IsolatedClientLoader.scala

```diff
@@ -103,7 +103,7 @@ private[hive] object IsolatedClientLoader extends Logging {
     case "2.2" | "2.2.0" => hive.v2_2
     case "2.3" | "2.3.0" | "2.3.1" | "2.3.2" | "2.3.3" | "2.3.4" | "2.3.5" | "2.3.6" => hive.v2_3
     case "3.0" | "3.0.0" => hive.v3_0
-    case "3.1" | "3.1.0" | "3.1.1" => hive.v3_1
+    case "3.1" | "3.1.0" | "3.1.1" | "3.1.2" => hive.v3_1
```

Comment:
Shall we update:

https://github.com/apache/spark/blob/02a0cdea13a5eebd27649a60d981de35156ba52c/sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveUtils.scala#L64-L67

and https://github.com/apache/spark/blob/02a0cdea13a5eebd27649a60d981de35156ba52c/docs/sql-data-sources-hive-tables.md#interacting-with-different-versions-of-hive-metastore 
?

Reply:
Oh, sorry. We also need to update https://github.com/apache/spark/blob/master/docs/sql-migration-guide-hive-compatibility.md